q2_fmt/tests/test_engraftment.py

```diff
@@ -1397,6 +1398,85 @@ def test_rename_features_with_blank_label(self):
         self.assertEqual("1", Fs1)
         self.assertEqual("2", Fs2)
 
+    def test_peds_nan_tp(self):
```

Comment:
Why is subject 2's sample dropped here?

Reply:
Because they don't have all the timepoints so I pass drop-incomplete subjects. I am going to refactor this  snf not do this filtering because I dont like this parameter (I have a big refactor that I am going to start when this PR is merged.)